plugins/device/init.go

```diff
@@ -27,6 +27,13 @@ func (p *Plugin) RestRoutes() route.Routes {
 			RequestType:  utils.GetTypeString((*deviceapi.AddDeviceReq)(nil)),
 			ResponseType: utils.GetTypeString((*deviceapi.AddDeviceResp)(nil)),
 			HandlerFunc:  deviceAddHandler},
+		route.Route{
```

Comment:
`endpoints.md`  need to be regenerated. 
* when we can have e2e test and CLI for this API change?

Reply:
I will add e2e, this URL I added since I added Location header in Add device handler